lib/govuk_publishing_components/presenters/shared_helper.rb

```diff
@@ -31,6 +31,23 @@ def get_heading_level
         "span"
       end
 
+      def get_heading_size(option, fallback)
+        govuk_class = "govuk-heading-"
+
+        case option
```

Comment:
Not critical but you could condense this down

```suggestion
        if ["xl", "l", "m", "s"].include? option
            "#{govuk_class}#{option}"
        else
            "#{govuk_class}#{fallback}"
        end
```

Reply:
Updated.